From: Carlos Henrique Lima Melara <carlos.melara@toradex.com>
Date: Fri, 2 Feb 2024 11:25:57 -0300
Subject: change default toolchain to gnu

---
 .cargo/config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.cargo/config b/.cargo/config
index 82a7d22..675a2c7 100644
--- a/.cargo/config
+++ b/.cargo/config
@@ -1,5 +1,5 @@
 [build]
-target = "x86_64-unknown-linux-musl"
+target = "x86_64-unknown-linux-gnu"
 
 [target.aarch64-unknown-linux-musl]
 linker = "aarch64-linux-musl-gcc"
